src/main/java/Duke.java

```diff
@@ -1,10 +1,261 @@
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+//TODO: specify error: empty description vs event/ddl timing not found
+//TODO: check print new line for printtask
+
 public class Duke {
+    public static String line = "____________________________________________________________\n";
+    //    public static int countTask = 0;
+//    public static Task[] tasks = new Task[100];
```

Comment:
Consider removing these lines of code if you no longer plan on/require them, in order to improve code readability.